src/main/java/pages/AlertPage.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
+
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.confirm;
+
+public class AlertPage {
+
+    public static void checkJSAlert() {
+        WebElement button = $(byText("Click for JS Alert"));
+        $(button).click();
+        confirm("I am a JS Alert");
+    }
+
+    public static void checkJSConfirm() {
+        WebElement button = $(byText("Click for JS Confirm"));
+        $(button).click();
+        confirm("I am a JS Confirm");
```

Comment:
As far as I remember this text will be on UI, not in alert, so this code will not work.